packages/elements/src/configuration/__test__/configuration.test.js

```diff
@@ -0,0 +1,27 @@
+import { expect, fixture, nextFrame } from '@refinitiv-ui/test-helpers';
+import { tickSvgBase64, tickSvg, isEqualSvg } from './helpers/helpers.js'
+import '@refinitiv-ui/elements/configuration';
+import '@refinitiv-ui/elements/icon';
+import '@refinitiv-ui/elemental-theme/light/ef-icon.js';
+
+describe('configuration/Configuration', () => {
```

Comment:
Didn't we agree to move the tests in the icon tests suite?
Also, we should add at least one test covering for the icon rendering without being wrapped in the context. That will make sure we are not breaking the functionality for users not implementing ef-configuration

Reply:
is it too late to disagree? :) I rather feel that all configuration test should set in configuration. Otherwise, when we do more config in the future, the configuration test will be scattered all over the place.

Reply:
It is not too late, don't worry
The logic these tests cover live in the icon component, so related tests should be placed there. If we keep tests here, when a bug is introduced in icon a test in a different test suite will fail, when the logic changes the developer will need to reach for tests in a different component to update etc.
I think what makes people think these tests should be placed here is just the title. We should just create a new describe block in the icon test suite

Reply:
Is it too late for me to explain? 55555
@Tolomeo we had discussed this on the Bankkok site and agreed to move these test cases under the icon element.
But for the configuration element, we think it should have some test case as well.

Reply:
Yep. My bad :) Thanks for Atom for clarification.